source/_integrations/ecobee.markdown

```diff
@@ -13,7 +13,7 @@ ha_release: 0.9
 ha_iot_class: Cloud Push
 ---
 
-The `ecobee` integration lets you control a thermostats and view sensor data from [ecobee](https://ecobee.com) thermostats.
+The `ecobee` integration lets you control a thermostats and view sensor data from [ecobee](https://www.ecobee.com) thermostats.
```

Comment:
The current link seems fine. The web server is responding with a HTTP 301 message.

```bash
$ curl -v https://ecobee.com/
[...]
> GET / HTTP/1.1
> Host: ecobee.com
[...]
< HTTP/1.1 301 Moved Permanently
< Date: Sat, 05 Oct 2019 16:33:48 GMT
< Location: https://www.ecobee.com/
[...]
```

Reply:
I ran xenu when i posted this and it came back as 'no connection' ...i couldn't access it via ecobee.com on the browser either, But had no issues with www.ecobee.com. Both seems to work now though and redirects like you said